src/crimson/os/seastore/seastore_types.h

```diff
@@ -0,0 +1,189 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <limits>
+#include <iostream>
+
+#include "include/denc.h"
+#include "include/buffer.h"
+#include "include/cmp.h"
+
+namespace crimson::os::seastore {
+
+using checksum_t = uint32_t;
+
+// Identifies segment location on disk, see SegmentManager,
+// may be negative for relative offsets
```

Comment:
"may be negative" for a uint32_t?

Reply:
Misplaced, applies to segment_off_t, I must have shuffled the definitions at some point.